packages/Form/Input/select-multi/src/MultiSelect.js

```diff
@@ -67,7 +68,7 @@ const MultiSelect = props => {
     <SelectComponent
       multi={false}
       name={name}
-      value={newValue}
+      value={newValue || ''}
```

Comment:
Do this operation few lines before, just when you create the newValue var.
```javascript
  const newValue = options.find(v => v.value === value)||'';
```

Reply:
Done